server/tableNamespaceClassifier.go

```diff
@@ -0,0 +1,46 @@
+package server
+
+import (
+	"github.com/pingcap/pd/server/core"
+	"github.com/pingcap/pd/server/namespace"
+)
+
+type tableNamespaceClassifier struct {
+	nsInfo *namespacesInfo
```

Comment:
Where is `namespaceInfo` defined? Forget to submit?

Reply:
It's in another PR https://github.com/pingcap/pd/pull/747

Saving/Loading namespace and classifier are separated.

Reply:
This PR is supposed to merge into `dantin/namespace`